docs/cugraph/source/tutorials/cugraph_notebooks.md

```diff
@@ -55,7 +55,7 @@ Running the example in these notebooks requires:
   * Download via Docker, Conda (See [__Getting Started__](https://rapids.ai/start.html))
 
 * cuGraph is dependent on the latest version of cuDF.  Please install all components of RAPIDS
-* Python 3.8+
+* Python 3.10+
 * A system with an NVIDIA GPU:  Pascal architecture or better
 * CUDA 11.4+
 * NVIDIA driver 450.51+
```

Comment:
We've been trying to purge specific driver references from our docs.
```suggestion
```

Reply:
sure! Will keep that in mind in the future.

I pushed https://github.com/rapidsai/cugraph/pull/4637/commits/02b2bbb3ab2d7c882aaaa2195e282d7c1d60e926 updating that wholegraph reference, found one other one with `git grep -i driver` too.